test/fixtures/test-runner/invalid_tap.js

```diff
@@ -0,0 +1 @@
+console.log('invalid tap output');
```

Comment:
add newline where needed

Reply:
can you elaborate?

Reply:
github expects files to end with an empty line

Reply:
Gotcha 👍